### ISSUE TITLE
add a language tag to DOCX exported document

### ISSUE THREAD

**Issue by Erwan-le-Gall** [suitenumerique/docs#1134]
label: accessibility

When I create a Docs document

Then I should be able to choose the langage of the document OR the default should be (fr-FR) French-France (and not en-NL English of Netherlands %-( ).

Why is this important: because disabled persons (blind…) use a screen reader and voice synthesis. Pronunciation is guided by this value. Ie the word "mobile" has a different pronunciation in English and in French.

**Comment by Erwan-le-Gall** [comment]
ping @cyberbaloo

**Comment by Ovgodd** [comment]
Hi,

I created an issue on BlockNote.JS [here](https://github.com/TypeCellOS/BlockNote/issues/1896) because the lang attribute is hardcoded and we don’t have access to change it.

It would be great to make it configurable.

Thanks!

**Comment by Ovgodd** [comment]
Hello ! I have create a PR on BlockNote.JS [here](https://github.com/TypeCellOS/BlockNote/pull/1937), to have the possiblity to make it configurable

**Comment by Erwan-le-Gall** [comment]
GG Cyril @Ovgodd

**Comment by cyberbaloo** [comment]
Well done! I tested it and it works well.

**Comment by Ovgodd** [comment]
My pleasure ;)
